lib/index.js

```diff
@@ -1,15 +1,89 @@
+var $ = require('jquery')
 var Food = require('./foods');
 var Exercise = require('./exercises');
 
 var someFood = new Food();
 var someExercise = new Exercise();
 
+setDiaryExercisesTable = function () {
```

Comment:
I like the exposure to doing this with vanilla JS.
I think we should refactor this into a Filter class at some point.